frontend/src/App.tsx

```diff
@@ -13,7 +13,7 @@ function App(): JSX.Element {
   };
 
   return (
-    <div className="flex h-screen bg-bg-dark text-white">
+    <div className="flex-wrap h-screen bg-bg-dark text-white md:flex">
```

Comment:
`md` is the medium breakpoint on tailwind
see docs: https://tailwindcss.com/docs/responsive-design